### PR TITLE
Fix PX4 mode bug

### PR DESCRIPTION
**Pull request by sanderux**

### CODE REVIEW

**Comment by sanderux** [comment]
@tridge could you have a look at this?

File: mavutil.py [hunk 1]
```diff
@@ -1580,9 +1580,9 @@ def interpret_px4_mode(base_mode, custom_mode):
             return "POSCTL"
     elif (base_mode & auto_mode_flags) == auto_mode_flags: #auto modes
         if custom_main_mode & PX4_CUSTOM_MAIN_MODE_AUTO != 0:
-            if custom_sub_mode == PX4_CUSTOM_SUB_MODE_AUTO_MISSION:
+	    if custom_sub_mode == PX4_CUSTOM_SUB_MODE_AUTO_MISSION:
```
Comment (lucasdemarchi) [inline mavutil.py]: your fix here is wrong... you are adding tabs instead of using only spaces

**Comment by lucasdemarchi** [comment]
You are adding tabs on the first commit and "fixing" it on the second by adding more.  Python sources here use only spaces. I fixed that and applied the first commit.

**Comment by sanderux** [comment]
@lucasdemarchi yeah i noticed QT was being a bit difficult replacing them at save. thanks!

**Comment by sanderux** [comment]
@lucasdemarchi any plans for a version bump and a pypi update? we would like to get dronekit compatible  but this issue makes it a bit hard t work with.

**Pull request closed without merging**